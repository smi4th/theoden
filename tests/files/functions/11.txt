fonction func(n) {
    res = 0;
    if (n<=1) {
        res = 1;
    }
    return res;
}

v = func(10);