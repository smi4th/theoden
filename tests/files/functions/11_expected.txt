---TEST OUTPUT---
---MEMORY---
v = 0
---FUNCTIONS---
func = (('param', 'n'), ('linst', ('linst', ('linst', ('assign', 'res', 0), 'empty'), ('if', ('<=', 'n', 1), ('linst', ('assign', 'res', 1), 'empty'), 'empty')), ('return', 'res')))
